### PR TITLE
add element.destroy to stripe-v3

### PR DESCRIPTION
**Pull request by benstevens48**
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Reference](https://stripe.com/docs/stripe-js/reference#other-methods)

This is my first pull request, so sorry if  I've done something wrong.

### CODE REVIEW

File: types/stripe-v3/index.d.ts [hunk 1]
```diff
@@ -185,6 +185,7 @@ declare namespace stripe {
             blur(): void;
             clear(): void;
             unmount(): void;
+            destroy(): void;
```
Comment (amritk) [inline types/stripe-v3/index.d.ts]: @benstevens48 looks good, but could you add a test?

**Review by amritk — CHANGES_REQUESTED** [review]

**Comment by benstevens48** [comment]
Ok, I added a test although I wasn't 100% sure about how the tests worked. As I understand they are just type-checked and not actually run. Also I wasn't quite sure why there were tests for stripe-v2 inside stripe-v3.

**Comment by amritk** [comment]
@benstevens48 the rests are run. Honestly I can't remember now, but here is the pull request where the versions were split https://github.com/DefinitelyTyped/DefinitelyTyped/pull/16539. Maybe they can be removed.

**Review by amritk — APPROVED** [review]

**Comment by amritk** [comment]
:+1:

**Comment by benstevens48** [comment]
Thanks. According to the DefinitelyTyped docs, the code in the tests is not actually run, only type-checked - [https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package). I hope this is the case else the `card.destroy()` I added to the test is called too early.

There are a few more updates I'd like to add if I have time (relating to 3D secure payment) but I'll open another pull request for that (if I have time).

**Comment by amritk** [comment]
Maybe that is the case then

**Comment by armanio123** [comment]
Test are not run only type-checked as @benstevens48 correctly mentioned.

**Pull request merged**